[2023-12-07T00:08:05.043+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag2.download_data scheduled__2023-08-20T00:00:00+00:00 [queued]>
[2023-12-07T00:08:05.049+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag2.download_data scheduled__2023-08-20T00:00:00+00:00 [queued]>
[2023-12-07T00:08:05.049+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-07T00:08:05.220+0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): download_data> on 2023-08-20 00:00:00+00:00
[2023-12-07T00:08:05.223+0500] {standard_task_runner.py:57} INFO - Started process 32085 to run task
[2023-12-07T00:08:05.226+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag2', 'download_data', 'scheduled__2023-08-20T00:00:00+00:00', '--job-id', '858', '--raw', '--subdir', 'DAGS_FOLDER/airflow_dag.py', '--cfg-path', '/tmp/tmp5qmbu7hu']
[2023-12-07T00:08:05.227+0500] {standard_task_runner.py:85} INFO - Job 858: Subtask download_data
[2023-12-07T00:08:05.437+0500] {task_command.py:416} INFO - Running <TaskInstance: etl_dag2.download_data scheduled__2023-08-20T00:00:00+00:00 [running]> on host amina-HP-ProBook-450-G2
[2023-12-07T00:08:05.863+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='amina_bibi' AIRFLOW_CTX_DAG_ID='etl_dag2' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-20T00:00:00+00:00'
[2023-12-07T00:08:05.869+0500] {logging_mixin.py:154} WARNING - /home/amina/miniconda/lib/python3.11/site-packages/gdown/parse_url.py:44 UserWarning: You specified a Google Drive link that is not the correct link to download a file. You might want to try `--fuzzy` option or the following url: https://drive.google.com/uc?id=10iZbAEEw7t8rtaD39OiuE9qPB8eSF3OJ
[2023-12-07T00:08:06.829+0500] {logging_mixin.py:154} WARNING - Downloading...
[2023-12-07T00:08:06.830+0500] {logging_mixin.py:154} WARNING - From: https://drive.google.com/file/d/10iZbAEEw7t8rtaD39OiuE9qPB8eSF3OJ/view?usp=sharing
[2023-12-07T00:08:06.830+0500] {logging_mixin.py:154} WARNING - To: /home/amina/airflow/dags/data.csv
[2023-12-07T00:08:06.832+0500] {logging_mixin.py:154} WARNING - 0.00B [00:00, ?B/s]
[2023-12-07T00:08:06.833+0500] {logging_mixin.py:154} WARNING - 6.86kB [00:00, 70.2MB/s]
[2023-12-07T00:08:06.834+0500] {python.py:194} INFO - Done. Returned value was: None
[2023-12-07T00:08:06.838+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_dag2, task_id=download_data, execution_date=20230820T000000, start_date=20231206T190805, end_date=20231206T190806
[2023-12-07T00:08:07.285+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-07T00:08:07.313+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
